packages/utils/hooks/useDebounce/index.ts

```diff
@@ -20,4 +20,21 @@ const useDebounce = <T extends (params: any) => any>(
   return { debouncedFunction: debouncedFuncRef.current }
```

Comment:
I also wanted to ask why we return this as an object instead of just the value?

Reply:
I don't think there was a real reason for that at the time, I guess because we were thinking about returning more data. I'm ok to return only the function instead